src/App.js

```diff
@@ -32,6 +33,7 @@ const App = () => (
           <Route exact path="/" component={Authenticate(Login)} />
           <Route exact path="/dashboard" component={Authenticate(UserDashboard)} />
           <Route exact path="/loginDetails" component={LoginDetails} />
+          <Route exact path="/signupform" component={Signup} />
```

Comment:
This loads a form but please use a better path name. Such as `signup`.

Reply:
+1 on this

Reply:
+1 on this

Reply:
@dondrzzy , @kmunene , @alexmochu  I made the requested changes but I had not respond to the comments